### PR TITLE
Optimize colour palette for Azure Pipelines

### PR DESCRIPTION
**Pull request by blairconrad**

### CODE REVIEW

**Comment by blairconrad** [comment]
Build with my changes: https://dev.azure.com/blairconrad/blair%20bullseye/_build/results?buildId=19
Build with original, rather sad, default palette: https://dev.azure.com/blairconrad/blair%20bullseye/_build/results?buildId=15

image

**Comment by blairconrad** [comment]
Bother. The help. Amended commit incoming.

**Comment by adamralph** [comment]
Thanks @blairconrad! It looks good, but this is weird:

50004177-4ee94600-ff74-11e8-8d53-c8b35a520468

I haven't seen this in any other build.

**Comment by blairconrad** [comment]
@adamralph, I think it's the `<>` in the output. I don't think it's technically a "palette" issue, but I can investigate!

**Comment by blairconrad** [comment]
Darnit. Running with `--no-colour` fixes it. I still blame the `<>`s, but am willing to call it a palette issue of some sort.

File: Bullseye/Internal/Logger.cs [hunk 1]
```diff
@@ -271,6 +271,7 @@ private static string ToStringFromMilliseconds(double milliseconds)
  {p.Option}-s, --skip-dependencies    {p.Text}Do not run targets' dependencies
  {p.Option}-v, --verbose              {p.Text}Enable verbose output
  {p.Option}    --appveyor             {p.Text}Force Appveyor mode (normally auto-detected)
+ {p.Option}    --azure-pipelines      {p.Text}Force Azure Piplines mode (normally auto-detected)
```
[the remark this answers is not in the document]
Comment (blairconrad) [reply Bullseye/Internal/Logger.cs]: Oops. Good eye.
Well, if the change is incorporated, and someone builds on Azure Pipelines, and the output includes angly brackets, they may lose some of the output, as you point out. For my use case, it's not a problem, but there is the potential for it to cause people grief. They can always workaround with `--no-color`, I suppose.

**Comment by blairconrad** [comment]
I took a shortcut and just amended, rather than pushing the fixup commit.

**Comment by adamralph** [comment]
Thanks @blairconrad!

**Comment by blairconrad** [comment]
Thank you!

**Comment by adamralph** [comment]
@blairconrad apologies, I've just realised how long ago this was merged and is still unreleased. There are a couple more things I want to get into 2.4.0, but I've released [2.4.0-alpha.1](https://www.nuget.org/packages/Bullseye/2.4.0-alpha.1) containing this change, should you wish to "early" adopt it.

**Pull request merged**